pkgs/development/python-modules/datadiff/default.nix

```diff
@@ -2,24 +2,35 @@
 , buildPythonPackage
 , fetchPypi
 , nose
+, pythonOlder
+, six
 }:
 
 buildPythonPackage rec {
   pname = "datadiff";
-  version = "1.1.6";
+  version = "2.0.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
```

Comment:
Why?

Reply:
Such a fact should be made visible through e.g. the `python36` package, it is not related to `datadiff`.
I always set to whatever upstream claims it supports.